tn_api_pricing_booking_spec.yml

```
@@ -858,55 +1039,134 @@ components:
-                            example: "2019-07-07"
-                        departure_date: 
-                            description: Date on which the first flight of the trip leaves.
-                            type: string
-                            example: "2019-11-21"
-                        flights: 
-                            description: List of all flights segment IATA codes
-                            type: array
-                            example: ['YHZ-YOW', 'YOW-YYZ', 'YYZ-YHZ']
-                        customer_email: 
-                            description: Email address of customer
-                            type: string
-                            example: customer@email.com
-                        customer_name: 
-                            description: Name of the primary passenger
-                            type: string
-                            example: customer@email.com
-                        total_price: 
-                            description: Total price of the trip including all taxes, fees, and markup.
-                            type: float
-                            example: 399.96
-                        currency: 
-                            description: Currency in which the booking was made
+                        status:
+                            description: Status of booking
                             type: string
-                            example: CAD
-                        payment:       
-                            type: boolean
-                            description: Notes whether the booking included payment information. 
-                            example: true
+                            example: booked
+                            enum: [booked,ticketed,cancelled]
+
+    passenger:
+      title: Primary passenger
+      type: object
+      properties:
+        first_name:
+          description: First Name
+          type: string
+          example: "Andres"
+        last_name:
+          description: Last Name
+          type: string
+          example: "Collart"
+
+    passenger_detail:
+      title: Primary passenger
+      type: object
+      properties:
+        first_name:
+          description: First Name
+          type: string
+          example: "Andres"
+        last_name:
+          description: Last Name
+          type: string
+          example: "Collart"
+        gender:
+          description: Gender
+          type: string
+          example: "M"
+          enum: [F, M]
+        passport_number:
+          description: Passport Number *Optional*
+          type: string
+          example: "KDS3468"
+        passport_expiry_date:
+          description: Passport Expiry Date *Optional*
+          type: string
+          format: date
+          example: "2021-11-21"
+        nationality:
+          description: Nationality *Optional*
+          type: string
+          example: "Canada"
+        type:
```

```suggestion
        passenger_type:
```

So these match the booking request object